Assets/_Graphics/Shaders/Editor/Note.shader

```diff
@@ -260,15 +262,16 @@ Shader "Custom/Note"
 					surfaceData.emission, surfaceData.alpha);
 
 				float isTranslucent = UNITY_ACCESS_INSTANCED_PROP(Props, _AlwaysTranslucent);
+				float animation = UNITY_ACCESS_INSTANCED_PROP(Props, _AnimationSpawned);
 
 				float translucentAlpha = UNITY_ACCESS_INSTANCED_PROP(Props, _TranslucentAlpha);
 				float opaqueAlpha = UNITY_ACCESS_INSTANCED_PROP(Props, _OpaqueAlpha);
-				
-				float alpha = (isTranslucent >= 1 || IN.rotatedPos.w <= 0) ? translucentAlpha : opaqueAlpha;
+
+				float alpha = (animation < 1 && (isTranslucent >= 1 || IN.rotatedPos.w <= 0)) ? translucentAlpha : opaqueAlpha;
 
 				clip(isDithered(IN.ScreenPosition.xy / IN.ScreenPosition.w, alpha));
 
-				return color; // float4(inputData.bakedGI,1);
+				return color * 2; // float4(inputData.bakedGI,1);
```

Comment:
Whats the reason behind multiplying color output by 2?

Reply:
https://discord.com/channels/702230714585710602/892494185909870602/1121697679391596586

With arrows using the note shader (for dissolve), they looked very gray. Multiplying by 2 makes the arrows look white, and makes the notes slightly more vibrant.